core/src/main/java/nl/nn/adapterframework/scheduler/JobDef.java

```diff
@@ -1209,8 +1206,7 @@ public String getJmsRealm() {
 	}
 
 	@IbisDoc({"Optional Locker, to avoid parallel execution of the Job by multiple threads or servers. The Job is NOT executed when the lock cannot be obtained, " +
-				"e.g. in case another thread, may be in another server, holds the lock and does not release it in a timely manner. " +
-				"N.B. To retain the lock, even after the transaction of the Job rolled back, set Lockers transactionAttribute=\"RequiresNew\" and type=\"P\"; Otherwise, the Locker will join the transaction of the Job and/or the Lock will be released."})
```

Comment:
Is this no longer true?

Reply:
Job execution is no longer transactional. There is no transaction for locker's transaction to join